android/src/main/java/com/aerobotics/DjiMobile/DJIMedia.java

```diff
@@ -0,0 +1,129 @@
+package com.aerobotics.DjiMobile;
+
+import com.facebook.react.bridge.Arguments;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.bridge.Promise;
+import com.facebook.react.bridge.WritableArray;
+import com.facebook.react.bridge.WritableMap;
+import com.facebook.react.modules.core.DeviceEventManagerModule;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+import dji.common.camera.SettingsDefinitions;
+import dji.common.error.DJIError;
+import dji.common.util.CommonCallbacks;
+import dji.sdk.camera.Camera;
+import dji.sdk.media.DownloadListener;
+import dji.sdk.media.MediaManager;
+import dji.sdk.media.MediaFile;
+import dji.sdk.sdkmanager.DJISDKManager;
+import dji.sdk.products.Aircraft;
+import dji.sdk.base.BaseProduct;
+
+public class DJIMedia  {
+  private Promise promise;
+  private MediaManager.FileListState currentFileListState = MediaManager.FileListState.UNKNOWN;
+  private MediaManager mediaManager;
+  private Camera camera;
+  private List<MediaFile> mediaFileList = new ArrayList<MediaFile>();
+
+  private MediaManager.FileListStateListener updateFileListStateListener = new MediaManager.FileListStateListener() {
+    @Override
+    public void onFileListStateChange(MediaManager.FileListState state) {
+      currentFileListState = state;
+    }
+  };
+
+  DJIMedia(){
+
+  }
+
+  public void getFileList(final Promise promise, BaseProduct baseProduct) {
```

Comment:
Maybe this is a bit pedantic but you have two methods with the same name (getFileList) in this class which do two different things. Could you possibly refactor this to make it a bit more understandable for the user?